src/main/kotlin/org/rust/ide/annotator/fixes/AddMutableFix.kt

```diff
@@ -54,7 +55,8 @@ fun updateMutable(project: Project, binding: RsNamedElement, mutable: Boolean =
                 parameter.replace(newParameterExpr)
                 return
             }
-            val newPatBinding = RsPsiFactory(project).createPatBinding(binding.identifier.text, mutable)
+            val isRefBinding = (binding.bindingMode?.ref) != null
+            val newPatBinding = RsPsiFactory(project).createPatBinding(binding.identifier.text, mutable, isRefBinding)
```

Comment:
Let's use named parameters for boolean args here to be sure that we don't mix up them